home/static/css/mystyles.css

```diff
@@ -137,18 +106,18 @@ div.bar-container {
   font-size: 1.5em;
   padding-bottom: 5px;
 }
-.post-comments {
-  display: block;
-}
 .close-frame {
   position: absolute;
   right: 10px;
 }
 .close-frame a:link {
   text-decoration: none;
-  color:gray; 
+  color:gray;
 }
 .close-frame a:visited {
   text-decoration: none;
   color:gray;
-}
\ No newline at end of file
+}
+.feed-icon {
```

Comment:
This style isn't getting applied for some reason that isn't obvious to me (at least in Chrome). The logo is hugging the top of the screen. 

![screen shot 2015-11-09 at 10 08 45 pm](https://cloud.githubusercontent.com/assets/719590/11053377/8055df26-872e-11e5-97c3-2be062cb9c6d.png)